### DISCUSSION TITLE
Show Min Max Time - Version 3.4.0

### DISCUSSION THREAD

**Discussion by gand07** [Daveiano/weewx-wdc#219] [Q&A] [answered]
Hi
Thanks for update 3.4, I really like the new add ons, especially the new tables in All time statistics. 

I have one question though. In skin.conf I have:

    show_min_max_time_day = True
    show_min_max_time_yesterday = True
    show_min_max_time_week = True
    show_min_max_time_month = True

and the min/max time are shown fine, except when I choose a year/month. When I hoover over min/max, no date/time are shown.
Screenshot version 3.4
![3-4](https://github.com/Daveiano/weewx-wdc/assets/47734099/09fbc68a-b3fd-4303-a9f9-e87e582bba79)
Screenshot version 3.3
![3-3](https://github.com/Daveiano/weewx-wdc/assets/47734099/9943aa0c-00ca-4dab-827f-cc1f825c8b01)

 
What I'm I missing
Thanks
Gert

**Comment by Daveiano** [comment, marked as answer]
Hi Gert,

the year and month pages are not re-generated if the time they are displaying is over (default WeeWX behaviour, nothing to do with the skin). So eg. if you change something in skin.conf, this will NOT trigger the re-generation of eg. month-2018-03, because it's over. 

You need to remove all past year and month pages to have them re-generated with your changes to skin.conf.

Something like:

```
rm public_html/year*
rm public_html/month*
```

Should do it, after that trigger a new report run and the time should be shown. Please let me know if this helps!

Regards, David

**Reply by gand07** [reply]
Hi
Thanks, issue solved
Gert
